index.js

```diff
@@ -0,0 +1,21 @@
+document.addEventListener("DOMContentLoaded", () => {
+    axios.get("http://localhost:3000/").then(res => {
+    res.data.forEach(person => {
+        let container = document.querySelector('#container')
+        let male_card = document.querySelector('.div')
+        let female_card = document.querySelector('.div')
+         let personName = person.name.first + " " + person.name.last + ", " + person.gender
+        let h3 = document.createElement("h3")
+         let li = document.createElement("li")
+         li.innerText = personName
+
```

Comment:
For this function you can either add a generic neutral "picture" for each person, or you can have an if-else statement checking for the gender of the individual (and then assigning an appropriate picture).